geode-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSetCommandsFunctionExecutor.java

```diff
@@ -37,18 +37,22 @@ public RedisSetCommandsFunctionExecutor(
     super(helper);
   }
 
-  private RedisSet getRedisSet(ByteArrayWrapper key) {
-    return helper.getRedisSet(key);
+  private RedisSet getRedisSet(ByteArrayWrapper key, boolean updateStats) {
```

Comment:
This is probably a nit, but for consistency can this class be consistent with the other `*FunctionExecutor`s and either introduce an overloaded `getRedisSet` which updates stats. Or remove the other classes' duplicate `getRedisSet` and just have one method.

Reply:
I think I made the change that you suggested but I'm not positive that I understood this correctly